.eslintrc.json

```diff
@@ -5,7 +5,9 @@
         "project": "./tsconfig.json"
     },
     "rules": {
-        "prettier/prettier": "error",
+        "prettier/prettier": ["error", {
```

Comment:
why did you add this condition, did the default of eslint change?

Reply:
I changed the rule because Prettier v2 default for trailing commas changed and the eslint fails without this extra rule.
Although I find it strange since I explicitly set "trailingComma": "none" in .prettierrc.json